--- abi.ARCH.glibc	2014-10-14 01:05:22.152105244 +0200
+++ abi.ARCH.musl	2014-10-14 01:05:22.344105238 +0200
@@ -23 +23 @@
-x_regoff_t(int, size (*) [4], align (*) [4])
+x_regoff_t(long, size (*) [8], align (*) [8])
@@ -35 +35 @@
-x_tcp_info(tcp_info, size (*) [104], align (*) [4])
+x_tcp_info(tcp_info, size (*) [120], align (*) [8])
@@ -42 +42 @@
-x_u_quad_t(unsigned long, size (*) [8], align (*) [8])
+x_u_quad_t(unsigned long long, size (*) [8], align (*) [8])
@@ -70 +70 @@
-x___jmp_buf(long*, size (*) [64], align (*) [8])
+x___jmp_buf(unsigned long*, size (*) [64], align (*) [8])
@@ -96 +96 @@
-x_crypt_data(crypt_data, size (*) [131232], align (*) [8])
+x_crypt_data(crypt_data, size (*) [260], align (*) [4])
@@ -120 +120 @@
-x_fpregset_t(_libc_fpstate*, size (*) [8], align (*) [8])
+x_fpregset_t(_fpstate*, size (*) [8], align (*) [8])
@@ -129 +129 @@
-x_ntptimeval(ntptimeval, size (*) [72], align (*) [8])
+x_ntptimeval(ntptimeval, size (*) [32], align (*) [8])
@@ -133 +133 @@
-x_regmatch_t(regmatch_t, size (*) [8], align (*) [4])
+x_regmatch_t(regmatch_t, size (*) [16], align (*) [8])
@@ -162 +162 @@
-x_sched_param(sched_param, size (*) [4], align (*) [4])
+x_sched_param(sched_param, size (*) [48], align (*) [8])
@@ -185 +185 @@
-x_ether_header(ether_header, size (*) [14], align (*) [1])
+x_ether_header(ether_header, size (*) [14], align (*) [2])
@@ -190,2 +190,2 @@
-x_int_fast16_t(long, size (*) [8], align (*) [8])
-x_int_fast32_t(long, size (*) [8], align (*) [8])
+x_int_fast16_t(int, size (*) [4], align (*) [4])
+x_int_fast32_t(int, size (*) [4], align (*) [4])
@@ -228,2 +228,2 @@
-x_uint_fast16_t(unsigned long, size (*) [8], align (*) [8])
-x_uint_fast32_t(unsigned long, size (*) [8], align (*) [8])
+x_uint_fast16_t(unsigned int, size (*) [4], align (*) [4])
+x_uint_fast32_t(unsigned int, size (*) [4], align (*) [4])
@@ -268 +268 @@
-x_pthread_rwlockattr_t(pthread_rwlockattr_t, size (*) [8], align (*) [8])
+x_pthread_rwlockattr_t(pthread_rwlockattr_t, size (*) [8], align (*) [4])
@@ -283 +283 @@
-x_CODE(_code, size (*) [16], align (*) [8])
+x_CODE(CODE, size (*) [16], align (*) [8])
@@ -319 +319 @@
-x_sem_t(sem_t, size (*) [32], align (*) [8])
+x_sem_t(sem_t, size (*) [32], align (*) [4])
@@ -328 +328 @@
-x_utmpx(utmpx, size (*) [384], align (*) [4])
+x_utmpx(utmpx, size (*) [400], align (*) [8])
@@ -338 +338 @@
-x_ethhdr(ethhdr, size (*) [14], align (*) [1])
+x_ethhdr(ethhdr, size (*) [14], align (*) [2])
@@ -363 +363 @@
-x_quad_t(long, size (*) [8], align (*) [8])
+x_quad_t(long long, size (*) [8], align (*) [8])
@@ -365,2 +365,2 @@
-x_rlim_t(unsigned long, size (*) [8], align (*) [8])
-x_rusage(rusage, size (*) [144], align (*) [8])
+x_rlim_t(unsigned long long, size (*) [8], align (*) [8])
+x_rusage(rusage, size (*) [272], align (*) [8])
@@ -382 +382 @@
-x_cmsghdr(cmsghdr, size (*) [16], align (*) [8])
+x_cmsghdr(cmsghdr, size (*) [16], align (*) [4])
@@ -384 +384 @@
-x_fd_mask(long, size (*) [8], align (*) [8])
+x_fd_mask(unsigned long, size (*) [8], align (*) [8])
@@ -401 +401 @@
-x_lastlog(lastlog, size (*) [292], align (*) [4])
+x_lastlog(lastlog, size (*) [296], align (*) [8])
@@ -427 +427 @@
-x_sysinfo(sysinfo, size (*) [112], align (*) [8])
+x_sysinfo(sysinfo, size (*) [368], align (*) [8])
@@ -429 +429 @@
-x_tftphdr(tftphdr, size (*) [5], align (*) [1])
+x_tftphdr(tftphdr, size (*) [6], align (*) [2])
