--- data/sizeof.ARCH.glibc	2014-03-20 14:16:34.540412317 +0100
+++ data/sizeof.ARCH.musl	2014-03-20 14:16:34.544412336 +0100
@@ -222,3 +222,3 @@
 struct cmsghdr	12
-struct crypt_data	131228
+struct crypt_data	260
 struct dirent	276
@@ -324,3 +324,3 @@
 struct ns_tsig_key	2060
-struct ntptimeval	36
+struct ntptimeval	16
 struct option	16
@@ -339,4 +339,4 @@
 struct rtentry	84
-struct rusage	72
-struct sched_param	4
+struct rusage	136
+struct sched_param	28
 struct sembuf	6
@@ -375,3 +375,3 @@
 struct strrecvfd	20
-struct sysinfo	64
+struct sysinfo	312
 struct tcp_info	104
@@ -380,3 +380,3 @@
 struct termios	60
-struct tftphdr	5
+struct tftphdr	6
 struct timeb	12
