--- data/glibc.sizeof	2014-03-11 11:37:53.000000000 +0100
+++ data/musl.sizeof	2014-03-11 11:37:53.000000000 +0100
@@ -111,4 +111,4 @@
 int8_t	1
-int_fast16_t	8
-int_fast32_t	8
+int_fast16_t	4
+int_fast32_t	4
 int_fast64_t	8
@@ -182,4 +182,4 @@
 register_t	8
-regmatch_t	8
-regoff_t	4
+regmatch_t	16
+regoff_t	8
 res_state	8
@@ -223,3 +223,3 @@
 struct cmsghdr	16
-struct crypt_data	131232
+struct crypt_data	260
 struct dirent	280
@@ -295,3 +295,3 @@
 struct itimerval	32
-struct lastlog	292
+struct lastlog	296
 struct lconv	96
@@ -325,3 +325,3 @@
 struct ns_tsig_key	2072
-struct ntptimeval	72
+struct ntptimeval	32
 struct option	32
@@ -340,4 +340,4 @@
 struct rtentry	120
-struct rusage	144
-struct sched_param	4
+struct rusage	272
+struct sched_param	48
 struct sembuf	6
@@ -376,3 +376,3 @@
 struct strrecvfd	20
-struct sysinfo	112
+struct sysinfo	368
 struct tcp_info	104
@@ -381,3 +381,3 @@
 struct termios	60
-struct tftphdr	5
+struct tftphdr	6
 struct timeb	16
@@ -396,3 +396,3 @@
 struct utimbuf	16
-struct utmpx	384
+struct utmpx	400
 struct utsname	390
@@ -419,4 +419,4 @@
 uint8_t	1
-uint_fast16_t	8
-uint_fast32_t	8
+uint_fast16_t	4
+uint_fast32_t	4
 uint_fast64_t	8
